tests/cases/compiler/functionsWithImplicitReturnTypeAssignableToUndefined.ts

```diff
@@ -0,0 +1,26 @@
+// @noImplicitReturns: false
+// @strictNullChecks: false

```

Comment:
FYI you can write:

```ts
// @strictNullChecks: true, false
```

To let the test runner test both cases for the same file.

Reply:
Oh, @DanielRosenwasser did use that [here](https://github.com/microsoft/TypeScript/issues/53473#issuecomment-1482016108) but I didn't know that was an actual feature. Neat.